_includes/releases/v22.2/v22.2.0-alpha.1.md

```diff
@@ -0,0 +1,851 @@
+- Active [Transactions](../{{site.versions["dev"]}}/ui-transactions-page.html) page no longer shows transactions from closed sessions. [#83896][#83896]
+- Fixed a bug that could cause an [optimizer](../{{site.versions["dev"]}}/cost-based-optimizer.html) panic in rare cases when a query had a left join in the input of an inner join. [#83875][#83875]
+- `DROP SCHEMA ... CASCADE` in the legacy schema changer now correctly fails when a [backup](../{{site.versions["dev"]}}/backup.html), [restore](../{{site.versions["dev"]}}/restore.html), or an [import](../{{site.versions["dev"]}}/import.html) of an underlying table or type is concurrently underway. [#84189][#84189]
+- `DROP ... CASCADE` of a database or a schema in the [declarative schema changer](../{{site.versions["dev"]}}/online-schema-changes.html#declarative-schema-changer) now correctly fails when a when a [backup](../{{site.versions["dev"]}}/backup.html), [restore](../{{site.versions["dev"]}}/restore.html), or an [import](../{{site.versions["dev"]}}/import.html) of an underlying table or type is concurrently underway. [#84189][#84189]
+- Fixed a bug that caused internal errors in rare cases when performing `DELETE`s on a table that had [foreign key](../{{site.versions["dev"]}}/foreign-key.html) references to it with the `ON DELETE CASCADE` option. For example, imagine tables `a` and `b` already exist, and `b` has a foreign key `ON DELETE CASCADE` column referencing `a`. If table `c` is added with a foreign key `ON DELETE CASCADE` column referencing table `b` and a `DELETE` statement is performed on table `a` in the same transaction, and internal error could occur. This bug has been present since v21.1.0. [#84219][#84219]
+- Fixed a bug that could corrupt [indexes](../{{site.versions["dev"]}}/indexes.html) and cause incorrect query results with `INTERVAL` values greater than about `290` years or less than about `-290` years. [#84045][#84045]
+- Fixed a bug that led to the `querySummary` field in the [`crdb_internal.statements_statistics`](../{{site.versions["dev"]}}/crdb-internal.html) metadata column being empty. [#84170][#84170]
+- Previously, CockroachDB could deadlock when evaluating analytical queries if multiple queries had to spill to disk at the same time. This is now fixed by making some of the queries error out instead. [#84398][#84398]
+- Fixed a bug where an ephemeral I/O error could crash a node. [#84449][#84449]
+- This PR fixed a bug where, in a [`ALTER PRIMARY KEY`](../{{site.versions["dev"]}}/alter-primary-key.html), if the new primary key columns is a subset of the previous primary key columns, CockroachDB will not rewrite existing secondary indexes, and hence those secondary indexes continue to have some of the previous primary key columns in their `suffixColumns`. But the user might, reasonably think those columns are not used anymore and proceed to drop them. The bug then caused those dependent secondary indexes to be dropped, unexpectedly for the user. [#84303][#84303]
```

Comment:
```
Fixed a bug where, in an [`ALTER PRIMARY KEY`](../{{site.versions["dev"]}}/alter-primary-key.html) statement, if the new primary key columns is a subset of the previous primary key columns, CockroachDB would not rewrite existing secondary indexes, and hence those secondary indexes continue to have some of the previous primary key columns in their `suffixColumns`.
```

I have no idea what to do with the rest of it.

Reply:
I found the language somewhat odd, but having thought it through communicates the idea clear enough, so left as-is